model/ftn/ww3_grid.ftn

```diff
@@ -523,6 +523,18 @@
                           NOSWLL, PTMETH, PTFCUT
       USE W3TIMEMD, ONLY: CALTYPE
       USE W3NMLGRIDMD
+!/SCRIP      USE SCRIP_GRIDS, ONLY: GRID1_UNITS, GRID1_NAME, &
+!/SCRIP                             GRID1_CENTER_LON, GRID1_CENTER_LAT, &
+!/SCRIP                             GRID1_CORNER_LON, GRID1_CORNER_LAT, &
+!/SCRIP                             GRID1_MASK, GRID1_SIZE, GRID1_RANK, &
+!/SCRIP                             GRID1_IMASK, &
+!/SCRIP                             GRID1_CORNERS, GRID1_DIMS
+!!/SCRIP      USE SCRIP_REMAP_VARS
```

Comment:
Remove commented out lines here and below?

Reply:
@sbrus89 could you take care of Jessica's comments about outlines?